src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.lucene.search.recipe.json

```diff
@@ -73,6 +73,12 @@
           ]
         }
       ]
+    },
+    {
+      "name": "lucene-index-rebuild",
```

Comment:
Executed this recipe after a fresh setup. Waited >3 minutes. Still, the index is empty and queries return no items. Logs are empty.

![image](https://user-images.githubusercontent.com/1976647/189400373-499b6548-4347-47f5-8935-ab03e7c8fb68.png)

Reply:
fixed